xml/System.Security.Cryptography.X509Certificates/X509ChainPolicy.xml

```diff
@@ -350,7 +348,7 @@
         <ReturnType>System.TimeSpan</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the time span that elapsed during online revocation verification or downloading the certificate revocation list (CRL).</summary>
+        <summary>Gets or sets the maximum amount of time to be spent during online revocation verification or downloading the certificate revocation list (CRL). A value of <see cref="F:System.TimeSpan.Zero" /> means there are no limits.</summary>
```

Comment:
It would be best to make this fully qualified: \<xref:System.TimeSpan,Zero?displayProperty=nameWithType>